### PR TITLE
Replace nox-automation with nox

### PR DESCRIPTION
**Pull request by reyang**
One possible way for #652. @c24t

### CODE REVIEW

**Comment by c24t** [comment]
I agree that this is the better approach (vs #652) in the long term, but it's still causing failing builds now. What do you think about merging #652 now and merging this one once CI is fixed?

**Comment by reyang** [comment]
> I agree that this is the better approach (vs #652) in the long term, but it's still causing failing builds now. What do you think about merging #652 now and merging this one once CI is fixed?

Yes I agree. Let's unblock the CI first using #652.

File: .circleci/config.yml [hunk 1]
```diff
@@ -40,7 +40,8 @@ jobs:
       - run:
           name: Run tests - opencensus
           command: |
-            nox -f nox.py
+            pip install --upgrade nox
```
Comment (reyang) [inline .circleci/config.yml]: This is a workaround to the preinstalled nox. We don't need this once we move to Travis.

**Comment by reyang** [comment]
@c24t CI is fixed.

**Comment by reyang** [comment]
FYI @lzchen

**Pull request merged**